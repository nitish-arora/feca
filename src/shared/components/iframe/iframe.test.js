import React from 'react';
import IFrame from './iframe';
import renderer from 'react-test-renderer';

const src= "";
const height = "";
const width = "";

test('should match snapshot with hidden options', () => {
  const component = renderer.create(
    <IFrame src={src} height={height} width={width} />
  );
  
  expect(component.toJSON()).toMatchSnapshot();
});