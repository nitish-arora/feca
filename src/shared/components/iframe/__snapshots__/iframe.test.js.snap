// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot with hidden options 1`] = `
<iframe
  frameBorder="0"
  height="100%"
  src=""
  title="sidenav-outlet"
  width="100%"
/>
`;
